src/types.ts

```diff
@@ -35,3 +35,35 @@ export type LocationData = {
   name: string;
   timezone: string;
 };
+export type DailyWeather = {
+  date: string;
+  symbol: string;
+  symbolPhrase: string;
+  maxTemp: number;
+  minTemp: number;
+  maxFeelsLikeTemp: number;
+  minFeelsLikeTemp: number;
+  maxRelHumidity: number;
+  minRelHumidity: number;
+  maxDewPoint: number;
+  minDewPoint: number;
+  precipAccum: number;
+  snowAccum: number;
+  maxWindSpeed: number;
+  windDir: number;
+  maxWindGust: number;
+  precipProb: number;
+  cloudiness: number;
+  sunrise: string;
+  sunset: string;
+  sunriseEpoch: number;
+  sunsetEpoch: number;
+  moonrise: string;
+  moonset: string;
+  moonPhase: number;
+  uvIndex: number;
+  minVisibility: number;
+  pressure: number;
+  confidence: string;
+  solarRadiationSum: number;
```

Comment:
WeatherData and DailyWeather interface have some duplicative properties, please create one common interface for them with these properties